Pipfile.lock

```diff
@@ -1,20 +1,7 @@
 {
```

Comment:
I think you may need to rebase, as we no longer have Pipfile.lock checked in.

Reply:
Is there a reason we aren't using the lockfile?

Reply:
Yes. It's so that we can continue to use pyup.io for monitoring insecure and outdated packages. Unfortunately although pyup.io supports Pipfile.lock, it's not a complete solution and has issues with new dependencies being introuced and conflicts with Pipfile modifications.